### DISCUSSION TITLE
Discards

### DISCUSSION THREAD

**Discussion by shipmadison** [pfmc-assessments/shortspine_thornyhead_2023#6] [General]
Hi Discard Team!

Starting a thread for discussion here. 

@pyhernvann @JaneSullivan-NOAA

**Comment by shipmadison** [comment]
Hi all! 

I was wondering if we could put our heads together either this afternoon (sometime before 4 pm) or tomorrow after class to discuss discards, the GEMM data, and the next steps. 

The original data request has been rejected due to confidentiality issues with our requested strata. I've pasted the specific reply from Chantel below. I'd like to get an amended request in ASAP! 

Thanks all, chat soon.

I have confirmed that there are confidentiality issues in both the discard and biological data in WCGOP given the stratification requested for the data. The issues primarily arise when the data is split by state and into trawl and fixed gears. These data have to be processed and uncertainty bootstrapped prior to use, this combined with the confidentiality nature of the data, make this data set one that can not be provided in a disaggregated form for data exploration.

Please revisit your consideration of fleet structure for the model and let us know if there is an alternative stratification that we can provide. WCGOP produces a groundfish expanded multi-year mortality report (GEMM) based on the data they collect. I recommend looking at this along with the PacFIN landings and biological data to guide the selection of fleet structure. The GEMM product can be helpful to not only see what the expanded discard mortality estimates by gear type but also the proportion of the exploitation arising from the non-catch share and catch-share sectors. The GEMM data can be obtained using the [{nwfscSurvey::pull_gemm}](https://github.com/pfmc-assessments/nwfscSurvey/blob/main/R/pull_gemm.R) function.

**Reply by JaneSullivan-NOAA** [reply]
[Link to original data request](https://github.com/pfmc-assessments/nwfscDiscard/issues/5#issuecomment-1411195713)

**Comment by somros** [comment]
@shipmadison FWIW on the rex sole side of things we ran into the same problem with the discards. I submitted a new request with no gear split (only trawl, for rex sole it makes the most sense anyway) and requesting data as north (WA+OR) and south (CA). I'll keep you posted on where this goes.

**Reply by shipmadison** [reply]
Thanks, Albi! I just asked a clarifying question on the original request about this, and we'll see what they say. Let me know how it works out for the rex group!

**Reply by somros** [reply]
@shipmadison The reviewed request seems to have pushed us over the line.

**Comment by pyhernvann** [comment]
Thanks @shipmadison - Can't make it today but I will tomorrow :) 
And Thanks for updating the request 

Just a few thoughts - if I correctly understand the discussion thread with Chantal and Andi 
- We could ask [WA+OR] vs [CA] for NonTrawl up to 2019 (2020 2021 problematical for fixed gears)
- We could ask [WA+OR] vs [CA] for Trawl up to 2021
- We may extrapolate discards for fixed gears in 2020-2021 based on previous years (considering the same values as those observed on average over 2017-2019 for instance). This is what has been done in the 2019 assessment if I am correct when they didn't submit a new request and used the discard info from the 2013 assessment

**Comment by pyhernvann** [comment]
Hi team -
Just a note that I have pushed (1) WCGOP discards data (not confidential) and (2) code for plotting the catch rates / length distri / average individual weight.
Also, the discard data that Kiva sent us is only WCGOP-derived information. They didn't prepare the data from the 2 other datasets containing older years. If one of these two must be similar to the one used in 2013, the other must have been updated since then.

**Comment by pyhernvann** [comment]
[DATA - UPDATE]
Hi @JaneSullivan-NOAA @Ovec8hkin @shipmadison 
quick news/update:
- John Wallace has planned to work on the Pikitch data before the end of the week
- WCGOP: biological data include sex but only 4 out 27,000+ individuals have been sexed for Shortspine
- Nothing new about EDCP

**Reply by JaneSullivan-NOAA** [reply]
@pyhernvann thank you for the update! if only 4 out of 27K specimens are sexed, we should definitely assume the length comps are unsexed!

**Comment by pyhernvann** [comment]
[ERROR - Weight units from WCGOP]
Hi @JaneSullivan-NOAA @Ovec8hkin @shipmadison
Just a note that, while asking for complementary data, @somros got the info from Andi Stephens that the WCGOP weights were in pounds (not specified in the data). I have just updated the calculations and the graphs. When I first generated the graphs, I compared them with the graphs of the stock assessment report and they seemed similar. So my guess is that the previous stock assessment team did this mistake as well. This would explain the substantial underestimation of the average weights they got out of the model.
